### PR TITLE
doc: Add the limitation notes of search proxy

### PR DESCRIPTION
**Pull request by jwcesign**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
none

**Which issue(s) this PR fixes**:
Fixes #none

**Special notes for your reviewer**:
none

### CODE REVIEW

**Comment by jwcesign** [comment]
cc @XiShanYongYe-Chang for checking

File: docs/installation/install-binary.md [hunk 1]
```diff
@@ -1011,10 +1011,10 @@ roleRef:
   name: cluster-proxy-clusterrole
 subjects:
   - kind: User
-    name: admin  #这里为证书的用户名，此安装默认为此用户
+    name: admin  # The name should be the user of client certificate, the default value is admin
```
Comment (XiShanYongYe-Chang) [inline docs/installation/install-binary.md:1014]: The description of the default value is misleading.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~

File: docs/installation/install-binary.md (hunk 1 above)
Comment (jwcesign) [reply docs/installation/install-binary.md]: fixed

**Comment by jwcesign** [comment]
cc @RainbowMango @XiShanYongYe-Chang for checking again

**Review by RainbowMango — COMMENTED** [review]
Generally looks good to me.

I vendor can we have a chapter that tells how to manage the `/proxy` API access privilege, that might be more clear.

**Comment by jwcesign** [comment]
> I vendor can we have a chapter that tells how to manage the /proxy API access privilege, that might be more clear.

It's better to propose the cluster proxy with `*` first, and then we can have a clearer understanding of the user's requirements before proceeding.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm

**Review by RainbowMango — APPROVED** [review]
OK
/approve

**Pull request merged**